DNN-OpenCV-Classification-with-Java/DnnOpenCV.java

```diff
@@ -0,0 +1,122 @@
+import org.opencv.core.Core;
+import org.opencv.core.Mat;
+import org.opencv.core.Rect;
+import org.opencv.core.Scalar;
+import org.opencv.core.Size;
+import org.opencv.dnn.Net;
+import org.opencv.dnn.Dnn;
+import org.opencv.imgproc.Imgproc;
+import org.opencv.imgcodecs.Imgcodecs;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import org.opencv.core.CvType;
+
+
+public class DnnOpenCV {
+    private static final int TARGET_IMG_WIDTH = 224;
+    private static final int TARGET_IMG_HEIGHT = 224;
+
+    private static final double SCALE_FACTOR = 1 / 255.0;
+
+    private static final String IMAGENET_CLASSES = "imagenet_classes.txt";
+    private static final String MODEL_PATH = "models/pytorch_mobilenet.onnx";
+
+
+    public static ArrayList<String> getImgLabels(String imgLabelsFilePath) throws IOException {
+        ArrayList<String> imgLabels;
+        try (Stream<String> lines = Files.lines(Paths.get(imgLabelsFilePath))) {
+            imgLabels = lines.collect(Collectors.toCollection(ArrayList::new));
+        }
+        return imgLabels;
+    }
+
+    public static Mat centerCrop(Mat inputImage) {
+        int y1 = Math.round((inputImage.rows() - TARGET_IMG_HEIGHT) / 2);
+        int y2 = Math.round(y1 + TARGET_IMG_HEIGHT);
+        int x1 = Math.round((inputImage.cols() - TARGET_IMG_WIDTH) / 2);
+        int x2 = Math.round(x1 + TARGET_IMG_WIDTH);
+
+        Rect centerRect = new Rect(x1, y1, (x2 - x1), (y2 - y1));
+        Mat croppedImage = new Mat(inputImage, centerRect);
+
+        return croppedImage;
+    }
+
+    public static Mat getPreprocessedImage(String imagePath) {
+        // define mean and standard deviation
+        Scalar mean = new Scalar(0.485, 0.456, 0.406);
+        Scalar std = new Scalar(0.229, 0.224, 0.225);
```

Comment:
I propose to make `mean` and `std` global variables like input resolution with `static final ...`

Reply:
Changed the fields to ``static final``.